### ISSUE TITLE
Class Validator Peer Dependency Error

### ISSUE THREAD

**Issue by iamomiid** [19majkel94/class-transformer-validator#25]
This is my package.json dependencies:
```
"class-transformer-validator": "^0.8.0",
"class-validator": "^0.11.0",
```
and I'm getting yarn error:
`"class-transformer-validator#class-validator@^0.10.1" doesn't satisfy found match of "class-validator@0.11.1"`

**Comment by Xetatronics** [comment]
same issue here with npm:
`npm WARN class-transformer-validator@0.8.0 requires a peer of class-validator@^0.10.1 but none is installed. You must install peer dependencies yourself.`

This warning keeps displaying and pollutes the console, however everything is working fine.

**Comment by MichalLytek** [comment]
Should be fixed in `v0.9.0` 🚀
